src/main/java/gov/cms/madie/terminology/controller/VsacFhirTerminologyController.java

```diff
@@ -85,4 +86,13 @@ public ResponseEntity<Code> getCode(
     return ResponseEntity.ok()
         .body(fhirTerminologyService.retrieveCode(code, codeSystem, version, user.getApiKey()));
   }
+
+  @PostMapping(path = "/codesList", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Preferred: `/codes-list`

Reply:
thanks for the comment I have changed it to /codes